packages/react-dom/src/client/ReactDOMFiberTextarea.js

```diff
@@ -91,31 +91,7 @@ export function initWrapperState(element: Element, props: Object) {
 
   // Only bother fetching default value if we're going to use it
   if (initialValue == null) {
-    let defaultValue = props.defaultValue;
-    // TODO (yungsters): Remove support for children content in <textarea>.
-    let children = props.children;
-    if (children != null) {
-      if (__DEV__) {
-        warning(
```

Comment:
Maybe the warning should be kept (or converted into an error) in 17.0? I know that this is technically a breaking change that warns already, but having an explicit `use value instead of children` warning would avoid confusing debugging. Shouldn't be any perf impact if it's dev-only too, right?